packages/fiori-freestyle/templates/worklist/add/webapp/controller/Worklist.controller.js

```diff
@@ -100,7 +98,7 @@ sap.ui.define([
 				var sQuery = oEvent.getParameter("query");
 
 				if (sQuery && sQuery.length > 0) {
-					aTableSearchState = [new Filter("<%=template.settings.entity.idProperty%>", FilterOperator.Contains, sQuery)];
+					aTableSearchState = [new Filter("ID", FilterOperator.Contains, sQuery)];
```

Comment:
Shouldn't the property match the ID property of the chosen entity? Or is `ID` always valid no matter the entity.
If not, this also needs to be looked at in in 133